php-frontend/src/test/java/org/sonar/php/symbols/ClassSymbolIndexTest.java

```diff
@@ -237,7 +237,7 @@ private ClassSymbolIndex createSymbols(ClassSymbolData... data) {
   }
 
   private ClassSymbolIndex createSymbols(ProjectSymbolData projectData, ClassSymbolData... data) {
-    ClassSymbolIndex result = ClassSymbolIndex.create(new HashSet<>(Arrays.asList(data)), projectData);
+    ClassSymbolIndex result = ClassSymbolIndex.create(new ArrayList<>(Arrays.asList(data)), projectData);
```

Comment:
```suggestion
    ClassSymbolIndex result = ClassSymbolIndex.create(Arrays.asList(data), projectData);
```
The list can be immutable as it will not be changed afterward.